src/styles/components/_number_picker.sass

```diff
@@ -18,6 +18,11 @@
   position: relative
   margin-bottom: 5px
 
+  input:disabled
+    // to hide the increment/decrement buttons when disabled
+    -moz-appearance: textfield
+    background-color: #fff
```

Comment:
is this background color right? Quickly tested the branch and it looks ok. But the `background-color: #fff` is being overruled, so it actually does nothing, and if it would do something, I think it would look odd.